sdk/python/feast/feature_store.py

```diff
@@ -200,7 +207,10 @@ def refresh_registry(self):
         """
         registry_config = self.config.registry
         registry = Registry(
-            self.config.project, registry_config, repo_path=self.repo_path
+            self.config.project,
+            registry_config,
+            repo_path=self.repo_path,
+            auth_config=self.config.auth_config,
```

Comment:
nit pick - is it too much work to just pass the config? i see 3 params out of 4 are injected from `RepoConfig`. if it is too much work just ignore this comment.

Reply:
I vote for passing the required objetcs as is, instead of the whole `config` container. This may also simplify the UT mocks.